architecture-building-systems/CityEnergyAnalyst
# 3346 added detail on optimal district energy networks

Add a file detailing the properties of the thermal networks of the non-dominated solutions.

_EDIT:_ 

To test this branch:
1. Run the new optimisation script on a district of your choice, with the setting 'Advanced > generate-detailed-outputs' set to 'True'.
2. Check the detailed optimisation results by navigating to 'YOUR_CASE\outputs\data\optimization_new\DXS_XXX\Supply_system_operation_details'. You should now see two new files that were not generated before the introduction of this branch: 'NXXXX_annual_breakdown.csv' and 'network_performance.csv'.

'NXXXX_annual_breakdown.csv' will provide more information on a thermal network's supply system, like the annual consumption of specific energy carriers and their associated cost and Heat and GHG emissions.

 'network_performance.csv' will provide additional details on the piping of all of the thermal networks within a system, e.g. thermal losses, length of the network, etc.
